apps/docs/src/.vitepress/utils-browser.ts

```diff
@@ -10,7 +10,7 @@ export type UseCssVariableValueOptions = {
    *
    * @default document.documentElement
    */
-  element?: Ref<HTMLElement | undefined>;
+  element?: Readonly<ShallowRef<HTMLDivElement | null>>;
```

Comment:
```suggestion
  element?: Readonly<ShallowRef<HTMLElement | null>>;
```

The utility is not limited to divs :)